preview-site-src/general/browser-support.adoc

```diff
@@ -10,6 +10,19 @@ Use Anypoint Platform with the latest version of any supported browser:
 * Google Chrome
 * Mozilla Firefox
 * Safari
+[tabs]
```

Comment:
Since you already have this new tabs syntax, please take out the original list (lines 8-12)

Reply:
@cheungaryk fixed